Umbraco-StarterKit/Creating-Content/Set-Up-Footer/index.md

```diff
@@ -0,0 +1,76 @@
+---
+versionFrom: 8.0.0
+---
+
+# Set-up your footer
+
+It can be challenging to make a footer if you never tried it before, this guide will help step-by-step, to get a nice looking informative footer for your page.
```

Comment:
```suggestion
It can be challenging to set-up a footer if you never tried it before. This guide will help step-by-step, to get a nice looking and informative footer for your website.
```